### ISSUE TITLE
TODO: Add title

### ISSUE THREAD

**Issue by berndbausch** [koalaman/shellcheck#2255]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash--
#I am a comment
#I am also a comment

echo "Hello World!!"

df -kh | awk '0+$5>=75{print $0}'

```

#### Here's what shellcheck currently says:
```

```
#### Here's what I wanted or expected to see:

It should complain about the two dashes in the shebang.

**Comment by koalaman** [comment]
You're right that this would be a problem. I looked through 25000 shebangs and found that 136 contained `--`, but I did not find any that used it incorrectly, so it does not appear to be a common or systemic issue. Thanks for the suggestion!
